src/components/ErrorFallback.js

```diff
@@ -0,0 +1,16 @@
+const React = require('react');
+
+module.exports = () => {
+
+    return (
+        <>
+            <h3>Oops! Something went wrong.</h3>
+            <h4>
+                You can keep browsing or try{' '}
```

Comment:
`try{' '}` looks odd?

Reply:
pardon if it's just something I'm unaware of, heh!

Reply:
I need to add a space between `try` and the `<button />` element on the next line.  The way JSX whitespace works, this seemed like the most direct way to do it to me, but I agree it's kinda awkward.

Reply:
Strange how the braces issue doesn't complain about this but complains about the other, /shrug